### DISCUSSION TITLE
print the result of response

### DISCUSSION THREAD

**Discussion by bengoldenberg** [hyperjumptech/monika#741] [Q&A]
is there a way to print the result of the request i sent?
i want to debug why the alert was triggered but i dont find a way to see the restult.
when i execute the request in postman it works, but in the pod it doesnt

**Comment by sapiderman** [comment]
hi @bengoldenberg,

Sure, check out `monika  --logs` this will print all the logs. What can help your alerts be meaningful? You can have a more specific alert message, since you set it. Have something like `Oops, http status is {{ response.status }}} and NOT 200!` as opposed to a more generic `you've got an alert.`  Another thing you can probably use is turn on the `--verbose` flag.

Check the [Alert docs here](https://monika.hyperjump.tech/guides/alerts) for more ideas.

Hope that helps! Let us know if you have further info!

**Comment by sapiderman** [comment]
@bengoldenberg  we've looked further and we think you've got a good point. Working on the ticket here: https://github.com/hyperjumptech/monika/issues/779

Keep checking the releases for updates!
